docs/api-docs/headless/orders.mdx

```diff
@@ -9,135 +9,19 @@ This section explains how to create and manage orders for headless storefronts.
 
 ## Creating an order from a cart
 
-To successfully complete this section, you must first create a cart with a **redirect URL**. For instructions on how to create a cart, see the [Carts]() section of this tutorial.
+To successfully complete this section, you must create a cart with a **redirect URL**. For instructions on creating a cart, see the [Carts](/api-docs/storefronts/guide/carts) section of this guide.
 
 ### Adding a billing address
 
-You can change a cart to a checkout by adding a billing address. To add a checkout billing address, send a `POST` request to the [Add Checkout Billing Address](/docs/rest-management/checkouts/checkout-billing-address#add-checkout-billing-address) endpoint. Your `checkoutId` will correspond with the cart `id` returned in response to [Create a Cart](/docs/rest-management/carts/carts-single#create-a-cart).
-
-```http filename="Example request: Add checkout billing address" showLineNumbers copy
-POST https://api.bigcommerce.com/stores/{store_hash}/v3/checkouts/{checkoutId}/billing-address
-Accept: application/json
-Content-Type: application/json
-X-Auth-Token: {{ACCESS_TOKEN}}
-
-{
-  "first_name": "Jane",
-  "last_name": "Doe",
-  "email": "jane@example.com",
-  "address1": "123 Main Street",
-  "address2": "",
-  "city": "Austin",
-  "state_or_province": "Texas",
-  "state_or_province_code": "TX",
-  "country_code": "US",
-  "postal_code": "78751",
-  "phone": "1234567890"
-}
-```
+You can change a cart to a checkout by adding a billing address. To add a checkout billing address, send a `POST` request to the [Add Checkout Billing Address](/docs/rest-management/checkouts/checkout-billing-address#add-checkout-billing-address) endpoint. Your `checkoutId` will correspond with the cart `id` returned in response to [Create a Cart](/docs/rest-management/carts/carts-single#create-a-cart). See the [headless tutorial](/api-docs/storefronts/guide/tutorial) for an example request.
```

Comment:
Unable  to find .mdx file or link for [headless tutorial](/api-docs/storefronts/guide/tutorial) and got a 404 when checking the link. Please advise.